cpp/dolfin/fem/Form.h

```diff
@@ -72,6 +73,9 @@ class Form
   Form(const std::vector<std::shared_ptr<const function::FunctionSpace>>&
```

Comment:
Argument list is getting quite long - some documentation explaining the arguments would be helpful.